dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/process/ProcessServiceImpl.java

```diff
@@ -3169,10 +3169,11 @@ public String findConfigYamlByName(String clusterName) {
         if (Strings.isNullOrEmpty(clusterName)) {
             return null;
         }
-        QueryWrapper<K8s> nodeWrapper = new QueryWrapper<>();
-        nodeWrapper.eq("k8s_name", clusterName);
-        K8s k8s = k8sMapper.selectOne(nodeWrapper);
-        return k8s.getK8sConfig();
+
+        QueryWrapper<Cluster> nodeWrapper = new QueryWrapper<>();
+        nodeWrapper.eq("name", clusterName);
+        Cluster cluster = clusterMapper.selectOne(nodeWrapper);
+        return cluster.getConfig();
```

Comment:
It's better to write a sql rather than use `QueryWrapper`, and when the cluster is null here we might better return null rather than NPE?

Reply:
> It's better to write a sql rather than use `QueryWrapper`, and when the cluster is null here we might better return null rather than NPE?

Hi, @ruanwenjun , thanks for the comment.

1. I've changed `return cluster.getConfig();` to `return cluster == null ? null : cluster.getConfig();` to avoid the potential NPE.
2. As for writing a SQL rather than using `QueryWrapper`, maybe we could do it in another PR? Since `QueryWrapper` is also used in many other places. And this NPE bug will cause the user cannot submit k8s tasks.

Reply:
>As for writing a SQL rather than using QueryWrapper, maybe we could do it in another PR? Since QueryWrapper is also used in many other places. Maybe we should fix this bug first since this NPE bug will cause the user cannot submit k8s tasks.

Hi @rickchengx do you want to do it in another pr?